mp/src/game/shared/momentum/mom_shareddefs.h

```diff
@@ -185,6 +185,17 @@ const char* const g_szTimesStatusStrings[] = {
     "#MOM_API_UnauthFriendsList", // STATUS_UNAUTHORIZED_FRIENDS_LIST
 };
 
+enum LocalTimesStatus_t
+{
+    LOCALREPLAY_READY,               // Everything was loaded correctly and is ready to be read
+    LOCALREPLAY_INPROGRESS,          // Filesystem is still busy
+    LOCALREPLAY_CANCELLED,               // The request was manually cancelled
+    LOCALREPLAY_BADREQUEST,               // The request was ill-formed (non-existent PathID)
+    LOCALREPLAY_FS_ERROR    // Every file failed due to a filesystem error
+};
+
+DECLARE_POINTER_HANDLE(MomReplayAsyncHandle);
+
```

Comment:
Should move into an ifdef CLIENT_DLL inside of the replay factory as this is only used by that factory, and therefore anything using it would be including it. Shareddefs is more like, project-globals typically.